static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -621,6 +623,7 @@ define([
     function shouldRenderLabel($adSlot) {
         return !$adSlot.hasClass('ad-slot--frame') &&
```

Comment:
Is there a better way to communicate whether a component handles its own labelling? We are having to swap between different classes on the same advert just to control this behaviour.